facade-test/createVM_test.go

```diff
@@ -0,0 +1,344 @@
+package facadetest
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/joho/godotenv"
+	prismgoclient "github.com/nutanix-cloud-native/prism-go-client"
+	prismclientv4facade "github.com/nutanix-cloud-native/prism-go-client/facade/v4"
+	"github.com/nutanix-cloud-native/prism-go-client/utils"
+	prismclientv3 "github.com/nutanix-cloud-native/prism-go-client/v3"
+	clustermgmtconfig "github.com/nutanix/ntnx-api-golang-clients/clustermgmt-go-client/v4/models/clustermgmt/v4/config"
+	vmmconfig "github.com/nutanix/ntnx-api-golang-clients/vmm-go-client/v4/models/vmm/v4/ahv/config"
+	vmmcontent "github.com/nutanix/ntnx-api-golang-clients/vmm-go-client/v4/models/vmm/v4/content"
+	"k8s.io/utils/ptr"
+)
+
+func GetCredentials() prismgoclient.Credentials {
+	err := godotenv.Load("../../prism-client-dotenv")
+	if err != nil {
+		fmt.Println("Error loading .env file: ", err)
+	}
+	username := os.Getenv("NUTANIX_USER")
+	password := os.Getenv("NUTANIX_PASSWORD")
+	endpoint := os.Getenv("NUTANIX_ENDPOINT")
+	return prismgoclient.Credentials{
+		Endpoint:    endpoint,
+		URL:         fmt.Sprintf("%s:9440", endpoint),
+		Username:    username,
+		Password:    password,
+		SessionAuth: true,
+		Insecure:    true,
+	}
+}
```

Comment:
no need to reinvent here: see https://github.com/nutanix-cloud-native/prism-go-client/blob/main/internal/testhelpers/creds.go